doc/lsp.txt

```diff
@@ -1001,6 +1001,13 @@ can use the following:
 >
     highlight LspDiagVirtualText ctermfg=Cyan guifg=Blue
 <
+
+or
+
+>
+    hi! link LspDiagLine NONE
+    hi! link LspDiagVirtualText WarningMsg
```

Comment:
Can you expand "hi" to "highlight"?  Also, the "!" is not needed.

Reply:
add fix in https://github.com/yegappan/lsp/pull/200/commits/a2c6afd1283991f8fd336112eb7d4df1b06ab9b1